lib/ace/ext/papeeria/tex_completer.coffee

```diff
@@ -84,16 +92,73 @@ define( (require, exports, module) ->
     value: word
     meta: 'equation'
   )
+  istype = (token, type) -> 
```

Comment:
any comments? What is token is expected to be, what is type and why such implementation? If `token.type == "werqewrqwerqwerqewrFOO"` then `isType(toke, "FOO") == true` -- is it expected and correct?